src/views/EventPage.vue

```diff
@@ -0,0 +1,266 @@
+<template>
+  <div class="page-container">
+    <!-- background img -->
+    <div class="background-img">
+      <div class="gradient-overlay" />
+      <img :src="event.mediaLink.cover" alt class="background-img" />
+    </div>
+
+    <NavBtns v-bind="navOptions" class="nav-btns" />
+    <!-- <ExploreBar /> -->
+
+    <!-- event details -->
+    <div v-rellax="{ speed: 10 }" class="event-container">
+      <!-- gallery -->
+      <div class="gallery">
+        <div
+          v-for="(img, index) in activeImages"
+          :key="`tag-${index}`"
+          class="image-container"
+        >
+          <img :src="img" alt />
+        </div>
+        <div v-if="event.mediaLink.host.length > 3" class="image-container">
+          <h4>+{{ event.mediaLink.host.length - 3 }}</h4>
+        </div>
+      </div>
+
+      <!-- tags -->
+      <div class="event-tags">
+        <div
+          v-for="(tag, index) in activeTags"
+          :key="`tag-${index}`"
+          class="event-tags__tag"
+        >
+          <h5>{{ tag }}</h5>
```

Comment:
tags have different font than normal

Reply:
hmm? it's meant to be source sans pro

Reply:
In view they're displaying as serif font somehow ya..

Reply:
oh tf lol can u check if the font is imported in urs

Reply:
Maybe not because I'm on a mac - should probably include a .ttf or something ya if it isn't a native font?

Reply:
ohh idk that might slow down performance? i'm just importing them thru google fonts import lol

Reply:
wait i just checked and source sans was not imported?? i'm pretty sure it was because it worked on mine lol

Reply:
Oh true right, importing google fonts is definitely better if its a free font - it might be working for you cuz you have it installed on your laptop by default